pkgs/development/python-modules/linode-cli/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, buildPythonApplication
+, fetchPypi
+, terminaltables
+, colorclass
+, requests
+, pyyaml
+, enum34
+, setuptools
+}:
+
+buildPythonApplication rec {
```

Comment:
buildPythonApplication shouldn't be used in `python-modules`, this should probably be moved to pkgs/tools/virtualization/linode-cli/default.nix, the top-level attr should be move to `all-packages`, and should look like:
```
  linode-cli = python3Packages.callPackage ../tools/virtualization/linode-cli/default.nix { };
```